### DISCUSSION TITLE
Error for 3108b5040116 and 3108b5110103

### DISCUSSION THREAD

**Discussion by szczukot** [john30/ebusd#1401] [Q&A]
Many people have exactly this same erorrs me too): 
[update error] unable to parse poll-read bai Status16 from 3108b5040116 / 00: ERR: invalid position
[update error] unable to parse poll-read bai Status from 3108b5110103 / 00: ERR: invalid position

What exactly is it and how to fix it (what to add to the Vaillant configuration)?

**Comment by dietersiebs** [comment]
Can you please write the message definition out of the csv file to this error?

**Reply by szczukot** [reply]
Sorry, I don't understand :(
What exactly am I supposed to do?

**Comment by dietersiebs** [comment]
Are you using the repository directly, or are you using local csv files with the definitions? I think, you use the files directly out of the repository in your configuration.

In the defition files are the definitions of the message of the ebus, so the ebusd could "translate" them. 

The defitions in the files looks like this:

*r,,,,,,"B511",,,,,,,,,,,,,,,,,,,,,,,,,
r,,State,,,,,09,,,UCH,,,,HeatDay,,energy,10,kWh,,Status2,,UIN,,,,OpMode,,UIN,10, ,

The definition of the two messages from you aren't correct. This could be caused by different things: maybe the data type, maybe the ID is not exact, maybe the count of the parts are not correct. Without this line out of the definitions csv, it ist not possible to say, whats wrong.

The structure of the definitions is described here:

https://github.com/john30/ebusd/wiki/4.1.-Message-definition

In your case the defitions of the bus messages bai "Status16" for 3108b5040116 and bai "Status" 3108b5110103 aren't correct and must be adapted. I think, you have to configure your system to hold the definition files locally, so you can edit/adapt them.

**Reply by szczukot** [reply]
I'm using configuration directly from repository.
I need to find B504 and B511 in the configuration.
How can i check, which file I use?

Do I have to look in this directory:
https://upd.ebusd.eu/
?

> Log
> > ebusd --foreground --mqtthost=core-mosquitto --mqttport=1883 --mqttuser=addons --mqttpass=zohMaiVieQu0eruk9uengeevah6ohsie8oopuefegae8Ai8hah9AiGh0phooY2on --scanconfig --mqttjson --mqttint=/etc/ebusd/mqtt-hassio.cfg --mqttvar=filter-direction=r|u|^w --mqtttopic=ebusd --device=ens:192.168.1.3:9999 --log=all:notice --log=update:error
> 2024-12-03 13:04:52.108 [main notice] ebusd 23.2.p20230716 started with auto scan on device: 192.168.1.3:9999, enhanced
> 2024-12-03 13:04:52.904 [bus notice] device status: resetting
> 2024-12-03 13:04:52.904 [bus notice] bus started with own address 31/36
> 2024-12-03 13:04:52.907 [bus notice] device status: reset, supports info
> 2024-12-03 13:04:52.910 [bus notice] device status: extra info: firmware 1.1[4a1b].1[4a1b], jumpers 0x0b
> 2024-12-03 13:04:52.922 [bus notice] signal acquired
> 2024-12-03 13:04:53.400 [mqtt notice] connection established
> 2024-12-03 13:04:53.412 [bus notice] new master 10, master count 2
> 2024-12-03 13:04:53.757 [bus notice] new master 03, master count 3
> 2024-12-03 13:05:03.103 [bus notice] scan 08: ;Vaillant;BAI00;0518;7401
> 2024-12-03 13:05:03.103 [bus notice] scan 08: ;Vaillant;BAI00;0518;7401
**> 2024-12-03 13:05:03.961 [main notice] read common config file vaillant/scan.csv
> 2024-12-03 13:05:04.147 [main notice] read common config file vaillant/broadcast.csv
> 2024-12-03 13:05:04.314 [main notice] read common config file vaillant/general.csv
> 2024-12-03 13:05:04.529 [main notice] read scan config file vaillant/08.bai.csv for ID "bai00", SW0518, HW7401**
> 2024-12-03 13:05:05.878 [main notice] found messages: 202 (0 conditional on 26 conditions, 0 poll, 10 update)
> 2024-12-03 13:05:06.600 [bus notice] scan 08: ;21;10;40;0010003869;3100;005327;N7
> 2024-12-03 13:05:08.762 [bus notice] scan 15: ;Vaillant;F4300;0120;2102
> 2024-12-03 13:05:08.763 [bus notice] scan 15: ;Vaillant;F4300;0120;2102
> 2024-12-03 13:05:09.027 [bus notice] max. symbols per second: 110
> 2024-12-03 13:05:09.445 [bus notice] scan 15: ;21;10;33;0020028524;0907;005556;N6
**> 2024-12-03 13:05:09.846 [main notice] read scan config file vaillant/15.f43.csv for ID "f4300", SW0120, HW2102**
> 2024-12-03 13:05:10.932 [main notice] found messages: 415 (7 conditional on 30 conditions, 70 poll, 10 update)
> 2024-12-03 13:05:13.115 [bus notice] scan 50: ;Vaillant;V6100;0119;1902
> 2024-12-03 13:05:13.115 [bus notice] scan 50: ;Vaillant;V6100;0119;1902
> 2024-12-03 13:05:13.897 [bus notice] scan 50: ;??;??;??;??????????;????;??????;??
> 2024-12-03 13:05:14.019 [bus notice] max. symbols per second: 139
**> 2024-12-03 13:05:14.257 [main notice] read scan config file vaillant/50.v61.mc.csv for ID "v6100", SW0119, HW1902**
> 2024-12-03 13:05:15.047 [main notice] found messages: 523 (11 conditional on 33 conditions, 70 poll, 10 update)

**Reply by szczukot** [reply]
I understand that I need to analyze these files

1)
https://github.com/john30/ebusd-configuration/blob/0d875e33573ff6545bba3577365cc0d8a16e798b/ebusd-2.1.x/en/vaillant/08.bai.csv
I see redirect to (for HW 7401)
https://github.com/john30/ebusd-configuration/blob/0d875e33573ff6545bba3577365cc0d8a16e798b/ebusd-2.1.x/en/vaillant/bai.308523.inc

2)
https://github.com/john30/ebusd-configuration/blob/0d875e33573ff6545bba3577365cc0d8a16e798b/ebusd-2.1.x/en/vaillant/15.f43.csv

3)
https://github.com/john30/ebusd-configuration/blob/0d875e33573ff6545bba3577365cc0d8a16e798b/ebusd-2.1.x/en/vaillant/50.v61.mc.csv

But I don't see B504 and B511 in any of these files.
